ACM-9979: Remove or make a note about using hypershift CLI (#6155)
open-cluster-management/rhacm-docs: clusters/hosted_control_planes/deploy_aws_private_clusters.adoc

hypershift create bastion aws --aws-creds=$AWS_CREDS --infra-id=$INFRA_ID --region=$REGION --ssh-key-file=$SSH_KEY
----

> *Note:* `hypershift` CLI is not available for download. Use the following commands to extract it from the hypershift operator pod. Replace `operator-747645c946-27w5k` with the actual operator pod name in `hypershift` namespace.

Reviewer: ```suggestion
*Note:* The `hypershift` CLI is not available to download. Use the following commands to extract it by using the HyperShift Operator pod present in the `hypershift` namespace. Replace `<hypershift-operator-pod-name>` with your HyperShift Operator pod name.
```

+
----

> oc project hypershift
> oc rsync operator-747645c946-27w5k:/usr/bin/hypershift-no-cgo .
> mv hypershift-no-cgo hypershift

Reviewer: ```suggestion
oc project hypershift
oc rsync <hypershift-operator-pod-name>:/usr/bin/hypershift-no-cgo .
mv hypershift-no-cgo hypershift
```
